main/src/ca/uwaterloo/flix/language/phase/jvm/GenRecordEmpty.scala

```diff
@@ -0,0 +1,137 @@
+package ca.uwaterloo.flix.language.phase.jvm
+
+import ca.uwaterloo.flix.api.Flix
+import ca.uwaterloo.flix.language.ast.FinalAst.Root
+import ca.uwaterloo.flix.language.ast.MonoType
+import org.objectweb.asm.ClassWriter
+import org.objectweb.asm.Opcodes._
+
+/**
+  * Generates bytecode for the tuple classes.
+  */
+object GenRecordEmpty {
+
+  /**
+    * Returns the set of tuple classes for the given set of types `ts`.
+    */
+  def gen(ts: Set[MonoType])(implicit root: Root, flix: Flix): Map[JvmName, JvmClass] = {
```

Comment:
The empty record is ALWAYS the same class, so we dont need to take the ts argument nor do we need to iterate through it and do all this stuff, right? We should just call genByteCode.